dev/com.ibm.ws.jdbc/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -110,9 +110,11 @@
   <AD id="databaseName"         required="false" type="String"  name="%databaseName" description="%databaseName.desc"/>
   <AD id="serverName"           required="false" type="String"  name="%serverName" description="%serverName.desc"/>
   <AD id="portNumber"           required="false" type="Integer" name="%portNumber" description="%portNumber.desc"/>
-  <AD id="URL"                  required="false" type="String"  name="%URL" description="%URL.desc"/>
+  <AD id="URL"                  required="false" type="String"  ibm:type="password" name="%URL" description="%URL.desc"/>
   <AD id="user"                 required="false" type="String"  name="%user" description="%user.desc"/>
   <AD id="password"             required="false" type="String"  ibm:type="password" name="%password" description="%password.desc"/>
+  <!-- If using generic properties and user provides oracle connectionProperties then make sure it is not traced as it could container passwords -->
+  <AD id="connectionProperties" required="false" type="String"  ibm:type="password" name="internal" description="internal use only"/>
```

Comment:
Changed `URL` to password and added `connectionProperties` as an internal element inside generic `properties` so that if someone is using generic properties to connect to oracle these passwords are still sufficiently hidden.